api/restapi/configure_keptn.go

```diff
@@ -102,11 +104,16 @@ func configureAPI(api *operations.KeptnAPI) http.Handler {
 	// api.EvaluationTriggerEvaluationHandler = evaluation.TriggerEvaluationHandlerFunc(handlers.TriggerEvaluationHandlerFunc)
 
 	// Import endpoint
+	importProcessor := importer.NewImportPackageProcessor(
+		new(model.YAMLManifestUnMarshaler), execute.NewKeptnExecutor(execute.StaticKeptnEndpointProvider{}),
+	)
+
 	api.ImportOperationsImportHandler = import_operations.ImportHandlerFunc(
 		handlers.GetImportHandlerFunc(
 			env.ImportBasePath,
-			importer.NewControlPlaneProjectChecker(os.Getenv(controlPlaneServiceEnvVar)),
+			handlers.NewControlPlaneProjectChecker(os.Getenv(controlPlaneServiceEnvVar)),
 			env.MaxImportUncompressedSize,
+			importProcessor,
```

Comment:
Please consolidate the two different ways of getitng the control plane URL here. Personally, I'd focus on the `os.Getenv` approach but within a function, so you don't repeat yourself.